### PR TITLE
python3.6

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

**Comment by antgonza** [comment]
This is longer than expected because:
- This was (one of or) the first plugin ever put together and interestingly it never was moved from python 2.7
- There were a lot of assumptions made that actually made it work
- This is now our first plugin as github actions

Anyway, is ready to be reviewed!

File: setup.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 from setuptools import setup
 from glob import glob
 
-__version__ = "0.1.0"
+__version__ = "2021.03"
 
 classes = """
     Development Status :: 3 - Alpha
```
Comment (ElDeveloper) [inline setup.py]: Is this still Alpha?

**Review by ElDeveloper — APPROVED** [review]
Looks good, just one comment about the "development status" of the project.

File: setup.py (hunk 1 above)
Comment (antgonza) [reply setup.py]: Excellent point, moving to: `Development Status :: 5 - Production/Stable` as all this have been deployed/in-used for a few year now.

**Pull request merged**